draft-specifications/CapTP Specification.md

```diff
@@ -0,0 +1,978 @@
+# Status of This Document
+
+This is a draft specification, created by the
+[OCapN](https://github.com/ocapn/ocapn) pre-standardization group. This draft is
+subject to change with the work of the group; if you're interested in being part
+of that work, please join!
+
+Authors: Jessica Tallon, Christine Lemmer-Webber, Mark Miller & The OCapN
+Pre-standardization Group.
+
+# [Introduction](#introduction)
+
+Capability Transport Protocol (CapTP) is a secure messaging protocol designed
+for communication between objects in distributed systems across networks. By
+utilizing a capability security model, CapTP ensures both security and
+functionality without compromise. In this model, an object can use a reference
+(capability) if (and only if) it possesses a reference to one. In other words,
+"if you don't have it, you can't use it."
+
+CapTP offers several valuable features, including:
+
+- Powerful promises and promise pipelining, allowing remote value usage before
+  resolution.
+- Error propagation across the network.
+- Distributed garbage collection.
+- Secure third-party handoffs, even when messages are published.
+
+CapTP is built upon the foundation of the actor model, where each actor is
+referred to as an object. An actor model is system where objects pass messages
```

Comment:
Should this be either "The actor model is a system where..." or "An actor model is a system where..."?